Description: The IPP backend did not send a cancel request to printers
 when a job was canceled and the printer did not support
 Create-Job.
Origin: http://svn.easysw.com/public/cups/trunk@10638
Date: Tue, 2 Oct 2012 16:30:35 +0000
Last-Update: 2012-11-06

--- a/backend/ipp.c
+++ b/backend/ipp.c
@@ -81,6 +81,7 @@
 #endif /* HAVE_GSSAPI && HAVE_XPC */
 static const char * const jattrs[] =	/* Job attributes we want */
 {
+  "job-id",
   "job-impressions-completed",
   "job-media-sheets-completed",
   "job-name",
